great_expectations/data_context/store/inline_store_backend.py

```diff
@@ -98,7 +101,12 @@ def _get(self, key: tuple[str, ...]) -> Any:
 
     @override
     def _get_all(self) -> list[Any]:
-        raise NotImplementedError
+        project_config = self._data_context.config
+        variable_config = project_config.get(self._resource_type)
+        if isinstance(variable_config, dict):
+            return list(variable_config.values())
+        else:
+            raise StoreBackendUnsupportedResourceTypeError(self._resource_type)
```

Comment:
I don't love this check, but it seems like kind of a necessity to guard against if the backend's resource type is something that doesn't make sense for us, like a store name.